keywords/LiteServNetMsft.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, version_build, host, port, storage_engine):
             "[windows:vars]",
             "ansible_user={}".format(os.environ["LITESERV_MSFT_HOST_USER"]),
             "ansible_password={}".format(os.environ["LITESERV_MSFT_HOST_PASSWORD"]),
-            "ansible_port=5986",
+            "ansible_port=5985",
```

Comment:
this port works only for new windows only. As long as we are running against the windows that is set up right now on jenkins, it is fine